### PR TITLE
interactive bots: Run the message_info bot for imangla

### PR DESCRIPTION
**Pull request by imangla**

### CODE REVIEW

**Comment by roberthoenig** [comment]
@imangla looking good so far! Two things:
* your commit "Run the helloworld bot" is included in this PR as well. Can you exclude it? (For the future, it's best to open new PRs on a separate branch instead of master; feel free to discuss this on chat.zulip.org with us :)

* your `notes.md` for the `message_info` bot is empty - if you didn't encounter any problems, maybe you can write what went especially easy when deploying the bots :)

**Comment by imangla** [comment]
Done!

File: interactive-bots/message_info/imangla/notes.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1. Initially, my message_info files were in the incorrect location so there were complications in running the bot. That was later sorted out.
```
Comment (tommyip) [inline interactive-bots/message_info/imangla/notes.md]: Can you line wrap text files at 79 characters.

**Comment by imangla** [comment]
Done!

**Pull request merged**